client.py

```diff
@@ -0,0 +1,65 @@
+import os
+import grpc
+from flask import Flask, jsonify, request, flash, request, redirect, url_for, abort, send_from_directory
+from google.protobuf.timestamp_pb2 import Timestamp
+from google.protobuf.json_format import MessageToJson
+import datetime
+import json
+
+
+#import the generated class
+import todo_pb2
+import todo_pb2_grpc
+
+# open a gRPC channel
+channel = grpc.insecure_channel('localhost:50051')
+
+# create a stub (client)
+stub = todo_pb2_grpc.TodoStub(channel)
+#App flask configuration
+
+app = Flask(__name__)
+
+#Add Todo
+@app.route('/addtodo', methods=['POST'])
+
+def add_todo():
+
+    title = request.form.get('title')
+    details = request.form.get('details')
+
+    deadline = Timestamp()
+    deadline.GetCurrentTime()
```

Comment:
why is the `deadline` defined by the server? The user should be passing the `deadline` in just like the `title` and `details`

Reply:
okay, I'll make the necessary corrections.